seatunnel-server/seatunnel-server-common/src/main/java/org/apache/seatunnel/server/common/SeatunnelErrorEnum.java

```diff
@@ -15,16 +15,29 @@
  * limitations under the License.
  */
 
-package org.apache.seatunnel.app.common;
+package org.apache.seatunnel.server.common;
 
 public enum SeatunnelErrorEnum {
 
     SCRIPT_ALREADY_EXIST(10001, "script already exist", "You already have a script with the same name : '%s'"),
     NO_SUCH_SCRIPT(10002, "no such script", "No such script. Maybe deleted by others."),
     USER_ALREADY_EXISTS(10003, "user already exist", "The same username [%s] is exist."),
     NO_SUCH_USER(10002, "no such user", "No such user. Maybe deleted by others."),
+    SCHEDULER_CONFIG_NOT_EXIST(10003, "scheduler config not exist", "This script's scheduler config not exist, please check your config."),
+    JSON_TRANSFORM_FAILED(10004, "json transform failed", "Json transform failed, it may be a bug."),
+
+    /**
+     * request dolphinscheduler failed
+     */
+    UNEXPECTED_RETURN_CODE(20000, "Unexpected return code", "Unexpected return code : [%s], error msg is [%s]"),
+    QUERY_PROJECT_CODE_FAILED(20001, "query project code failed", "Request ds for querying project code failed"),
+    NO_MATCHED_PROJECT(20002, "no matched project", "No matched project [%s], please check your configuration"),
+    NO_MATCHED_SCRIPT_SAVE_DIR(20003, "no matched script save dir", "No matched script save dir [%s], please check your configuration"),
+
+    UNSUPPORTED_OPERATION(99996, "unsupported operation", "This operation [%s] is not supported now."),
```

Comment:
Why the code begins with `99996` :).

Reply:
These are generic exceptions, decreasing from '99999‘.